core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -609,9 +606,6 @@ public void writeIndexingBuffer() throws EngineException {
                              new ByteSizeValue(indexingBufferBytes), new ByteSizeValue(versionMapBytes));
                 indexWriter.flush();
             }
-        } catch (AlreadyClosedException e) {
```

Comment:
Same comment here as above: do we still need to catch this and then immediately rethrow otherwise it will get eaten by the general catch block below?